python/ray/train/mosaic/mosaic_trainer.py

```diff
@@ -171,5 +180,12 @@ def _mosaic_train_loop_per_worker(config):
     config["progress_bar"] = False
     trainer: Trainer = trainer_init_per_worker(config)
 
+    # Remove Composer's Loggers
+    filtered_callbacks = list()
+    for callback in trainer.state.callbacks:
+        if not isinstance(callback, LoggerDestination):
+            filtered_callbacks.append(callback)
+    trainer.state.callbacks = filtered_callbacks
```

Comment:
nice! is there a way to add a test for this?

Maybe add a dummy `LoggerDestination` Mosaic Logger that raises an error, but make sure that training can still finish without error. That means the logger must have been removed.

Reply:
A test has been added!